src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        Integer resultOne = result1;
+        Integer resultTwo = result2;
+        Integer betOne = bet1;
+        Integer betTwo = bet2;
+        return (result1 - bet1 == 0 && result2 - bet2 == 0)
+                ? 2 : (resultOne.compareTo(resultTwo) == (betOne.compareTo(betTwo)) ? 1 : 0);
```

Comment:
try to use it without redundant variables https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Reply:
done